content/en/docs/concepts/configuration/configmap.md

```diff
@@ -16,33 +16,45 @@ or use additional (third party) tools to keep your data private.
 {{< /caution >}}
 
 
-
 <!-- body -->
 ## Motivation
 
 Use a ConfigMap for setting configuration data separately from application code.
 
 For example, imagine that you are developing an application that you can run on your
 own computer (for development) and in the cloud (to handle real traffic).
-You write the code to
-look in an environment variable named `DATABASE_HOST`. Locally, you set that variable
-to `localhost`. In the cloud, you set it to refer to a Kubernetes
-{{< glossary_tooltip text="Service" term_id="service" >}} that exposes the database
-component to your cluster.
-
+You write the code to look in an environment variable named `DATABASE_HOST`.
+Locally, you set that variable to `localhost`. In the cloud, you set it to
+refer to a Kubernetes {{< glossary_tooltip text="Service" term_id="service" >}}
+that exposes the database component to your cluster.
 This lets you fetch a container image running in the cloud and
 debug the exact same code locally if needed.
 
+A ConfigMap is not designed to hold large chunks of data. The data stored in a
+ConfigMap cannot exeed 1 MiB. If you need to store settings that are
+larger than this limit, you may want to consider mounting a volume or use a
+separate database or file service.
+
 ## ConfigMap object
 
 A ConfigMap is an API [object](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
 that lets you store configuration for other objects to use. Unlike most
-Kubernetes objects that have a `spec`, a ConfigMap has a `data` section to
-store items (keys) and their values.
+Kubernetes objects that have a `spec`, a ConfigMap has `data` field and
+`binaryData` fields. These fields accepts key-value pairs as their values.
```

Comment:
nit: should it be:
a ConfigMap has `data` and `binaryData` fields?

Reply:
That suggested wording looks better to me.

Reply:
yes, will revise.